src/main/java/net/atos/zac/app/personen/converter/RESTPersoonConverter.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * SPDX-FileCopyrightText: 2022 Atos
+ * SPDX-License-Identifier: EUPL-1.2+
+ */
+
+package net.atos.zac.app.personen.converter;
+
+import java.util.Objects;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+import org.apache.commons.lang3.StringUtils;
+
+import net.atos.client.brp.model.Geboorte;
+import net.atos.client.brp.model.IngeschrevenPersoonHal;
+import net.atos.client.brp.model.NaamPersoon;
+import net.atos.client.brp.model.Verblijfplaats;
+import net.atos.zac.app.personen.model.RESTPersoonOverzicht;
+
+public class RESTPersoonConverter {
+
+    public static final String FIELDS_PERSOON_OVERZICHT = "naam.voornamen,naam.voorvoegsel,naam.geslachtsnaam,geboorte.datum.datum," +
+            "verblijfplaats.straat,verblijfplaats.huisnummer,verblijfplaats.huisnummertoevoeging,verblijfplaats.huisletter,verblijfplaats.woonplaats";
+
+
+    private static final String ONBEKEND = "<onbekend>";
+
+    public RESTPersoonOverzicht convert(final IngeschrevenPersoonHal persoon) {
+        final RESTPersoonOverzicht persoonOverzicht = new RESTPersoonOverzicht();
+        if (persoon != null) {
+            persoonOverzicht.naam = convertTotNaam(persoon.getNaam());
+            persoonOverzicht.geboortedatum = convertToGeboortedatum(persoon.getGeboorte());
+            persoonOverzicht.inschrijfadres = convertToInschrijfadres(persoon.getVerblijfplaats());
+        } else {
+            persoonOverzicht.naam = ONBEKEND;
+            persoonOverzicht.geboortedatum = ONBEKEND;
+            persoonOverzicht.inschrijfadres = ONBEKEND;
+        }
+        return persoonOverzicht;
+    }
+
+    private String convertTotNaam(final NaamPersoon naam) {
```

Comment:
Gezien de andere 2 convert methoden convertToX heten, is convertTot waarschijnlijk een typo?